dist/i18n/GUI_Strings_en.txt

```diff
@@ -103,6 +103,7 @@ menu.help.lookUpInDictionary(F1) = Look Up In Dictionary (F1)
 menu.help.netLogoUserManual = NetLogo User Manual
 menu.help.netLogoDictionary = NetLogo Dictionary
 menu.help.netLogoUsersGroup = NetLogo Users Group
+menu.help.netLogoStackOverflow = NetLogo Q & A - StackOverflow
```

Comment:
I think it's better to leave it in. Informally, we call netlogo-users netlogo-users and the fact that it's on yahoo is incidental, but in the Stack Overflow case, it's not really its own thing, it's just "Stack Overflow". But not everybody knows what that is, so the "NetLogo Q&A" should be there too IMO.